source/includes/edgedelivery/_delivery_rules.md

```diff
@@ -139,16 +139,16 @@ curl -X PUT \
 }
 ```
 
-<code>PUT /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/deliveryrules/:ruleId?siteId=<a href="#stackpath-sites">:siteId</a></code>
+<code>PUT /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/deliveryrules/:ruleId?siteId=<a href="#edgedelivery-sites">:siteId</a></code>
 
 | Query Params        | &nbsp;                                                                                |
 | ------------------- | ------------------------------------------------------------------------------------- |
 | `siteId`<br/>_UUID_ | The ID of the site for which to update the delivery rule. This parameter is required. |
 
 | Required                                                    | &nbsp;                  |
 | ----------------------------------------------------------- | ----------------------- |
-| `conditions`<br/>_Array[[Condition](#stackpath-condition)]_ | At least one condition. |
-| `actions`<br/>_Array[[Action](#stackpath-action)]_          | At least one action.    |
+| `conditions`<br/>_Array[[Condition](#edgedelivery-condition)]_ | At least one condition. |
+| `actions`<br/>_Array[[Action](#edgedelivery-action)]_          | At least one action.    |
```

Comment:
Was this broken before your change?

Reply:
yes they were broken before since the folder name `stackpath` had been split to two:`edgecompute` and `edgedelivery, but the references were never updated

Reply:
Makes sense, thanks!